### ISSUE TITLE
make test complains about Comparator

### ISSUE THREAD

**Issue by AuroraBTH** [aurora-munge/databasteknik#17]
`make test` crashes almost immediately with the following error:

```
PHP Fatal error:  Declaration of SebastianBergmann\Comparator\DOMNodeComparator::assertEquals($expected, $actual, $delta = 0, $canonicalize = false, $ignoreCase = false) must be compatible with SebastianBergmann\Comparator\ObjectComparator::assertEquals($expected, $actual, $delta = 0, $canonicalize = false, $ignoreCase = false, array &$processed = Array) in phar:///Applications/XAMPP/xamppfiles/htdocs/dbwebb-kurser/databasteknik/.bin/phpunit/sebastian-comparator/DOMNodeComparator.php on line 110
PHP Stack trace:
PHP   1. {main}() /Applications/XAMPP/xamppfiles/htdocs/dbwebb-kurser/databasteknik/.bin/phpunit:0

Fatal error: Declaration of SebastianBergmann\Comparator\DOMNodeComparator::assertEquals($expected, $actual, $delta = 0, $canonicalize = false, $ignoreCase = false) must be compatible with SebastianBergmann\Comparator\ObjectComparator::assertEquals($expected, $actual, $delta = 0, $canonicalize = false, $ignoreCase = false, array &$processed = Array) in phar:///Applications/XAMPP/xamppfiles/htdocs/dbwebb-kurser/databasteknik/.bin/phpunit/sebastian-comparator/DOMNodeComparator.php on line 110

Call Stack:
    0.0056     536032   1. {main}() /Applications/XAMPP/xamppfiles/htdocs/dbwebb-kurser/databasteknik/.bin/phpunit:0

make: *** [phpunit] Error 255
```

Can most likely be replicated with ```make install && make test```.

OS: MacOS 10.13.3
PHP: 7.2.2
Composer: 1.5.1

**Comment by MagnusGreiff** [comment]
I can confirm this problem.  The same happens to me on:
OS: Ubuntu 16.04
PHP: 7.2.2
Composer: 1.6.3

**Comment by AuroraBTH** [comment]
Update to PHPUnit 7.0.0 is necessary.

**Comment by AuroraBTH** [comment]
Fixed with #25
